src/viewer/styles.styl

```diff
@@ -41,11 +41,25 @@
     align-items      center
     flex-direction   column
 
+    h2
+        max-width   90%
+
+.pho-viewer-imageviewer
+.pho-viewer-nav + .pho-viewer-pdfviewer
```

Comment:
I know this is legacy code but I guess there is something smarter to do here, like giving all those element a same class. Not blocking comment though.